### PR TITLE
Adding docassemble-cucumber package to use for testing

### LINKED ISSUE #139: Use question ids in cucumber tests
instead of 
Then I should see phrase "question text"
do
Then I should see question question-id

### PR DESCRIPTION
**Pull request by niharikasingh**
Updating the testing package, and making all the tests pass again. 

Once this is updated, think about whether this closes #139 as well. I think it might...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,32 +1,36 @@
 {
   "name": "docassemble-juvenilesealing",
-  "version": "1.0.0",
-  "description": "This process should be easier for both petitioners themselves and legal professionals who hope to support petitioners. This tool is focused on MA.",
-  "main": "index.js",
+  "version": "0.1.0",
+  "description": "",
+  "main": "",
```
Comment (niharikasingh) [inline package.json:5]: undo these changes to version and description

**Review by knod — APPROVED** [review]
It passes! It's amazing!
